src/Raven.Studio/typescript/models/database/status/storageReportItem.ts

```diff
@@ -8,6 +8,7 @@ class storageReportItem {
     type: string;
     internalChildren: storageReportItem[];
     size?: number;
+    length?: number;
```

Comment:
what this length represents?

Reply:
Length is used to calculate the length for '_stream_'
Needed for the `customSizeProvider` (inside method` mapStreams()`)

Reply:
Can't we use size?

Reply:
For streams, the server returns both `Length` & `AllocatedSpaceInBytes`
which is used here:
https://github.com/ravendb/ravendb/blob/7f32742ee1fe90c30b9d4242cdb3234a3b589268/src/Raven.Studio/typescript/viewmodels/database/status/storageReport.ts#L225